embedded-kafka/src/main/scala/net/manub/embeddedkafka/UUIDs.scala

```diff
@@ -0,0 +1,14 @@
+package net.manub.embeddedkafka
+
+import java.util.UUID
+
+/** Utility object for creating unique test IDs.
+  * Useful for separating IDs and directories across test cases.
+  */
+object UUIDs {
+  /** Create a new unique ID.
+    *
+    * @return the unique ID
+    */
+  def newUuid(): String = UUID.randomUUID().toString
```

Comment:
I'd rather have this method still returning a `UUID` and let the user convert it to string.